Nightly: App Crash on "Show as List"

**Describe the bug:**
When selecting "Show as List" from the Live Map Menu, c:geo immediately restarts

**To Reproduce:**
Steps to reproduce the behavior:
1. Open "Live Map" and move to a region with at least two caches
2. Click on "Show as List"
3. c:geo immediately restarts, ending up in start screen, logging in into geocaching,com

**Version of c:geo used:**
2019.09.20-NB-28ab651

**Is the problem reproducible:**
Yes

**System information:**
```
--- System information ---
Device: VOG-L29 (VOG-L29EEA, HUAWEI)
Android version: 9
Android build: VOG-L29 9.1.0.193(C431E4R2P2)
c:geo version: 2019.09.20-NB-28ab651
Google Play services: disabled - 19.2.75 (100400-269183835)
Low power mode: inactive
Compass capabilities: yes
Rotation vector sensor: present
Orientation sensor: present
Magnetometer & Accelerometer sensor: present
Direction sensor used: rotation vector
Hide own/found: true
HW acceleration: enabled (default state)
System language: de_DE
System date format: dd.MM.yy
Debug mode active: no
System internal c:geo dir: /data/user/0/cgeo.geocaching (31,4 GB free) internal
User storage c:geo dir: /storage/emulated/0/cgeo (31,4 GB free) external non-removable
Geocache data: /storage/emulated/0/Android/data/cgeo.geocaching/files/GeocacheData (31,4 GB free) external non-removable
Database: /data/user/0/cgeo.geocaching/databases/data (287,2 MB) on system internal storage
Fine location permission: granted
Write external storage permission: granted
Geocaching sites enabled:
   geocaching.com: Logged in (Anmeldung OK) / PREMIUM
   extremcaching.com: Logged in (Anmeldung OK)
Geocaching.com date format: dd MMM yy
Installed c:geo plugins: contacts
--- End of system information ---
```

**Additional context:**
I'm pretty sure this worked with the previous nightly (as I update frequently and use this feature a lot), thus it may be related to one of the latest changes.

Error is 
~~~
   java.lang.RuntimeException: Unable to start activity ComponentInfo{cgeo.geocaching/cgeo.geocaching.CacheListActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Class java.lang.Object.getClass()' on a null object reference
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2927)
...
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Class java.lang.Object.getClass()' on a null object reference
        at android.support.v4.app.LoaderManagerImpl.createAndInstallLoader(LoaderManagerImpl.java:371)
        at android.support.v4.app.LoaderManagerImpl.initLoader(LoaderManagerImpl.java:404)
        at cgeo.geocaching.CacheListActivity.onCreate(CacheListActivity.java:524)
        at android.app.Activity.performCreate(Activity.java:6942)
...
~~~

--- a/build.gradle
+++ b/build.gradle
@@ -155,7 +155,7 @@ buildScan {
  */
 
 ext {
-    supportLibraryVersion = '27.1.1'
+    supportLibraryVersion = '27.0.2'
 }
 
 subprojects {

